cmd/podman/machine/list.go

```diff
@@ -0,0 +1,142 @@
+// +build amd64,linux amd64,darwin arm64,darwin
```

Comment:
You have to add `arm64,linux`. At least the other commands have it.

Reply:
yup